pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -103,6 +97,12 @@ sub generate_advice {
 
                 }
             }
+
```

Comment:
What precisely is re-arranging the execution of this block accomplishing?

Reply:
We need to know if IAVP is licensed before we do the IAV suggestions.